app/controllers/birds_controller.rb

```diff
@@ -0,0 +1,10 @@
+class BirdsController < ApplicationController
+  def show
+    @birds = Trail.find(params[:id]).region.birds
+    render 'show.json.jbuilder'
+  end
+  def index
+    @birds = Bird.all
+    render 'index.json.jbuilder'
```

Comment:
By default, you don't need to do render 'index.json.jbuilder' for index, show, new and edit actions.